### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by clytaemnestra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# pyvo-bot
+
+A bot 🤖 which reminds people that the [Pyvo](https://pyvo.cz) meetup 🍻 is coming! 
+
+## Usage
+
+Currently, the bot supports only [Telegram](https://telegram.org/) and can be found at https://t.me/pyvo_bot. The bot runs once per day here using the CircleCI build. It loads iCalendar feed of Pyvo Praha and checks when is the next meetup. If it's about to be soon, it notifies the [Pyvo Praha](https://t.me/pyvopraha) Telegram channel.
```
Comment (honzajavorek) [inline README.md:7]:
```suggestion
Currently, the bot supports only [Telegram](https://telegram.org/) and can be found at https://t.me/pyvo_bot. The bot runs once per day here using the GitHub Actions build. It loads iCalendar feed of Pyvo Praha and checks when is the next meetup. If it's about to be soon, it notifies the [Pyvo Praha](https://t.me/pyvopraha) Telegram channel.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+```bash
+python3 main.py 
+```
+
+## To Do
+
+* refactor code (functions could be more decoupled)
+* write tests
+
+Contributions are welcomed.
```
Comment (honzajavorek) [inline README.md]:
```suggestion
Contributions are welcome. 
```

Comment (honzajavorek) [reply README.md]: Not an English expert, but I think this is more correct?

**Review by honzajavorek — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (honzajavorek) [reply README.md:7]: (At least I thought it's gonna run here on GitHub, but it's your choice)

File: README.md (hunk 2 above)
Comment (clytaemnestra) [reply README.md]: You're right! Both are grammatically correct and commonly used, but `welcome` is apparently more suitable for this situation.  

https://english.stackexchange.com/questions/2121/which-is-correct-feedback-is-welcome-or-feedback-is-welcomed

**Pull request merged**